src/index.test.ts

```diff
@@ -1,8 +1,8 @@
-import {describe, expect, test} from '@jest/globals'
-import axios from "axios"
+import { describe, expect, test } from '@jest/globals'
 import nock from 'nock'
-import {SpvWalletClient} from "./index";
-import {ClientOptions, DraftTransaction, Recipients, TransactionConfigInput} from "./interface";
+import fetchMock from "jest-fetch-mock"
```

Comment:
It just didn't work. I thought it would be possible to release this change on Friday and I didn't want to waste time working on the mock.
I don't mind revisiting this if you think it makes sense